www/src/pages/foundations/elevation.jsx

```diff
@@ -0,0 +1,269 @@
+            key={key}
+            min={key === 'x' || key === 'y' ? '-100' : '0'}
+            max="100"
+            type={key === 'color' ? 'color' : 'range'}
+            defaultValue="0"
+            onChange={(e) => updateBoxShadowModel(key, e.target.value)}
+          />
+        </Form.Label>
+      ))}
+      <Form.Checkbox
+        onChange={() => updateBoxShadowModel('inset', !boxShadowModel.inset)}
+        floatLabelLeft
+      >
+        Inset:
+      </Form.Checkbox>
+    </section>
+  );
+};
+
+BoxShadowToolkit.propTypes = {
+  updateBoxShadow: PropTypes.func.isRequired,
+  id: PropTypes.number.isRequired,
+};
+
+const BoxShadowGenerator = () => {
+  const [boxShadows, setBoxShadows] = useState([]);
+  const [controls, setControls] = useState([{ index: 0 }]);
+
+  const updateBoxShadow = (shadow, id) => {
+    const boxShadow = [...boxShadows];
+    boxShadow[id] = shadow.inset ? `inset ${shadow.x}px ${shadow.y}px ${shadow.blur}px ${shadow.spread}px ${shadow.color}`
+      : `${shadow.x}px ${shadow.y}px ${shadow.blur}px ${shadow.spread}px ${shadow.color}`;
+    setBoxShadows(boxShadow);
+  };
+
+  const addBoxShadow = () => {
+    setControls([...controls, {
+      index: controls.length,
+    }]);
+  };
+
+  return (
+    <section className="pgn-doc__box-shadow-generator">
+      <div className="pgn-doc__box-shadow-generator--preview">
+        <div
+          className="pgn-doc__box-shadow-generator--preview-box border"
+          style={{ boxShadow: boxShadows.join(',') }}
+        />
+      </div>
+      <div className="pgn-doc__box-shadow-generator--toolkit">
+        <div className="d-flex overflow-auto mb-2">
+          {controls.map(control => (
+            <BoxShadowToolkit
+              key={control.index}
+              id={control.index}
+              updateBoxShadow={updateBoxShadow}
+            />
+          ))}
+        </div>
+        <div className="d-flex justify-content-between flex-column flex-md-row">
+          <code className="pgn-doc__box-shadow-generator--toolkit-code mb-3 mb-md-0">
+            box-shadow: {boxShadows.join(', ')};
+          </code>
+          <Button onClick={addBoxShadow} variant="dark">Add New Layer</Button>
+        </div>
+      </div>
+    </section>
+  );
+};
+
+export default function ElevationPage() {
+  const levelTitle = boxShadowLevels.map(level => (
+    <h3 key={level} className="pgn-doc__box-shadow-level-title">
+      Level {level}
+    </h3>
+  ));
+
+  const sideTitle = boxShadowSides.map(side => (
+    <h3 key={side} className="pgn-doc__box-shadow-side-title">
+      {side.charAt(0).toUpperCase() + side.substring(1)}
+    </h3>
+  ));
+
+  return (
+    <Layout>
+      <Container className="py-5" size="md">
+        {/* eslint-disable-next-line react/jsx-pascal-case */}
+        <SEO title="Elevation" />
+        <h1 className="mb-3">Elevation & Shadow</h1>
+        <h2 className="pgn-doc__box-shadow--title">Clickable Box-Shadow Grid</h2>
```

Comment:
nit: I think this heading can be removed in favor of just having the text below the above `h1`.

Reply:
Good, removed